rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/core/RestControllerHelper.java

```diff
@@ -393,9 +393,12 @@ public Component updateComponent(Component componentToUpdate, Component requestB
     }
 
     public Release updateRelease(Release releaseToUpdate, Release requestBodyRelease) {
-        for(Release._Fields field:Release._Fields.values()) {
+        for (Release._Fields field : Release._Fields.values()) {
+            if (Release._Fields.CLEARING_STATE.equals(field)) {
```

Comment:
`Release` document have 30+ fields, So rather than adding an `if` condition that's being checked for each field in for loop, I would suggest to reset the `clearingState` field in controller class.. something like this: `updateRelease.setClearingState(sw360Release.getClearingState());`